.github/workflows/testing.yml

```diff
@@ -13,18 +13,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['8.1', '8.2']
-        drupal-core: ['10.2.x']
+        php-versions: ['8.1', '8.2', '8.3']
+        drupal-core: ['10.2.x', '10.3.x']
```

Comment:
i think it is enough to test 10.2.x just on PHP 8.1, and only test 10.3 on all the versions